src/features/ledger/DeviceScan.tsx

```diff
@@ -12,39 +11,35 @@ import {
   BottomSheetModalProvider,
   BottomSheetScrollView,
 } from '@gorhom/bottom-sheet'
-import BackButton from '@components/BackButton'
 import Text from '@components/Text'
 import Box from '@components/Box'
 import TouchableOpacityBox from '@components/TouchableOpacityBox'
 import { useColors, useOpacity } from '@config/theme/themeHooks'
 import useBackHandler from '@hooks/useBackHandler'
 import useLedgerDeviceScan from '@hooks/useLedgerDeviceScan'
+import { useOnboarding } from '@features/onboarding/OnboardingProvider'
+import { useOnboardingSheet } from '@features/onboarding/OnboardingSheet'
 import LedgerConnectSteps from './LedgerConnectSteps'
-import {
-  LedgerNavigatorNavigationProp,
-  LedgerNavigatorStackParamList,
-} from './ledgerNavigatorTypes'
 
-type Route = RouteProp<LedgerNavigatorStackParamList, 'DeviceScan'>
 const DeviceScan = () => {
   const { t } = useTranslation()
-  const navigation = useNavigation<LedgerNavigatorNavigationProp>()
-  const route = useRoute<Route>()
+  // const route = useRoute<Route>()
   const { primaryText } = useColors()
 
   const { backgroundStyle } = useOpacity('bg.tertiary', 1)
   const bottomSheetModalRef = useRef<BottomSheetModal>(null)
   const [contentHeight, setContentHeight] = useState(0)
   const { handleDismiss, setIsShowing } = useBackHandler(bottomSheetModalRef)
   const { refreshing, error, devices, setError, reload } = useLedgerDeviceScan()
+  const { setOnboardingData } = useOnboarding()
+  const { carouselRef } = useOnboardingSheet()
+  // useEffect(() => {
+  //   if (!route.params?.error) {
+  //     return
+  //   }
 
-  useEffect(() => {
-    if (!route.params?.error) {
-      return
-    }
-
-    setError(route.params.error)
-  }, [route, setError])
+  //   setError(route.params.error)
+  // }, [route, setError])
```

Comment:
why is this commented out?

Reply:
Removed. Not needed anymore since this page is not using params and now uses the onboarding provider